### ISSUE TITLE
Combined/merged sequence runs - best point to merge?

### ISSUE THREAD

**Issue by eeeaston** [faircloth-lab/phyluce#305]
I have multiple sequence runs for the same sample but different library preps.  I usually cat them prior to trimming, but when I ran the script with the illumiprocessor.conf file, it errors, so more than two tags and duplicate tag maps are not supported. At what step in the workflow would you suggest merging the outputs?

**Comment by brantfaircloth** [comment]
I would trim them separately, then cat together the correct resulting files before assembly (be sure to create directory format and filenames similar to what you would normally get from illumiprocessor).

**Comment by brantfaircloth** [comment]
Yep, that's pretty much it!  You want the directory to look like:
```
uce-tutorial
└──  clean-fastq
   └──  alligator_mississippiensis
      └──  split-adapter-quality-trimmed
         ├── alligator_mississippiensis-READ1.fastq.gz
         ├── alligator_mississippiensis-READ2.fastq.gz
         └── alligator_mississippiensis-READ-singleton.fastq.gz

```
